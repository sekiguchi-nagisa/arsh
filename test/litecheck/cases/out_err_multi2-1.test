# RUN: /bin/sh $self
echo 1234
# CHECK: 1234
echo 1>&2 ABCD
# CHECKERR: ABCD
echo 6789
# CHECK: 6789
# CHECK: 0123
echo 1>&2 EFGH
# CHECKERR: EFGH
echo 1>&2 IJKL

