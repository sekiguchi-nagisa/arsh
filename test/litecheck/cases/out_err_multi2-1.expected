3d2
< 0123
stdout is mismatched
2a3
> IJKL
stderr is mismatched